cli/src/ui.rs

```diff
@@ -375,14 +385,17 @@ impl Ui {
 
     /// Writer to print an update that's not part of the command's main output.
     pub fn status(&self) -> Box<dyn Write + '_> {
-        Box::new(self.stderr_formatter())
+        if self.quiet {
+            Box::new(PlainTextFormatter::new(std::io::sink()))
+        } else {
+            Box::new(self.stderr_formatter())
+        }
```

Comment:
nit: Formatter isn't needed here
```rust
if self.quiet { Box::new(io::sink()) } else { Box::new(self.stderr()) }
```

However, we'll need something similar in `hint_*()` methods.
```rust
self.status_formatter().unwrap_or_else(|| ..)
```